src/index.jsx

```diff
@@ -32,8 +37,10 @@ if (returnPath) {
   store.dispatch(push(returnPath));
 }
 
-store.dispatch(fetchAccounts());
-store.dispatch(fetchRoles());
+store.dispatch(fetchDatasets());
```

Comment:
Shouldn't a (for example) dataSetsContainer dispatch a fetchDatasets action instead of the index file?

Reply:
you are right but we need all datasets anyway